package.json

```diff
@@ -54,15 +55,26 @@
     "typescript-styled-plugin": "^0.2.0"
   },
   "devDependencies": {
+    "@types/jest": "^22.0.1",
     "@types/react": "^16.0.19",
     "@types/react-dom": "^16.0.2",
+    "babel-core": "^6.26.0",
+    "babel-jest": "^22.0.4",
```

Comment:
Shouldn't need to add this, it's built in to jest.

Reply:
Shouldn't actually need anything `babel-*` (including `.babelrc` as there's no JS code in the repo)

Reply:
I added it to get the JS tests working, since TS/JS were both not working.

Reply:
There's probably a poor caching profile to using both, I'd just go with TS for the tests as well.

Reply:
Unless you can get `@babel/preset-typesript` working

Reply:
I can't get TS working.

Reply:
Using the config in this PR.